package/CHANGELOG

```diff
@@ -107,6 +108,7 @@ Enhancements
     'protein' selection (#2751 PR #2755)
   * Added an RDKit converter that works for any input with all hydrogens
     explicit in the topology (Issue #2468, PR #2775)
+  * Improved performance of the ParmEd converter (Issue #3028, PR #3029)
```

Comment:
This should probably be under enhancements?
Also entries are ordered newer first :)